src/test/cpp/src/ctest_main.cc

```diff
@@ -24,6 +24,39 @@
 // Amortize the cost of building catch2.hpp for
 // every unit test by linking in ctest_main.o
 
-#define CATCH_CONFIG_MAIN
+//See https://github.com/catchorg/Catch2/blob/master/docs/own-main.md#adding-your-own-command-line-options
+//to pass run time arguments to tests
+
+#define CATCH_CONFIG_RUNNER
 #include <catch2/catch.hpp>
 
+std::string g_pb_query_json_file = "";
+
+int main( int argc, char* argv[] )
+{
+  Catch::Session session; // There must be exactly one instance
+
+  int height = 0; // Some user variable you want to be able to set
```

Comment:
Looks like cruft from the Catch2 docs. Is it needed here?

Reply:
Removed